ROOTFS/etc/waggle/nw/config.ini

```diff
@@ -5,6 +5,7 @@ check_successive_seconds = 5.0
 rssh_addrs = [ ('beehive', 'beehive', 20022), ('beekeeper', 'beehive.honeyhouse.one', 49190) ]
 network_services = [ "NetworkManager", "ModemManager", "waggle-reverse-tunnel", "waggle-bk-reverse-tunnel" ]
 sd_card_storage_loc = /media/scratch
+wifi_adapter_loc_file = /etc/waggle/nw/adapter_loc_file
```

Comment:
since its only valid for this boot you can use a tmpfs path like `/run/waggle/nw`